Themes/default/Agreement.template.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+/**
+ * Simple Machines Forum (SMF)
+ *
+ * @package SMF
+ * @author Simple Machines
+ * @copyright 2018 Simple Machines
+ * @license http://www.simplemachines.org/about/smf/license.php BSD
+ *
+ * @version SMF 2.1 RC2
+ */
+
+// The main sub template - show the agreement and/or privacy policy
+function template_main()
+{
+	global $context, $scripturl, $txt;
+
+	if (!empty($context['accept_doc']))
+		echo '
+	<form action="', $scripturl, '?action=acceptagreement;doc=', $context['accept_doc'], '" method="post">';
+
+	if (!empty($context['agreement']))
+	{
+		echo '
+		<div class="cat_bar">
+			<h3 class="catbg">', $txt['agreement' . ($context['can_accept_agreement'] ? '_updated' : '')], '</h3>
+		</div>';
+
+		if ($context['can_accept_agreement'])
+		{
+			echo '
+		<div class="description">
+			', $txt['agreement_updated_desc'], '
+		</div>';
+		}
+		elseif (!empty($context['agreement_accepted_date']))
+		{
+			echo '
+		<div class="description">
+			', sprintf($txt['agreement_accepted'], timeformat($context['agreement_accepted_date'], false)), '
+		</div>';
+		}
+
+		echo '
+		<div class="roundframe">
```

Comment:
I suggest both description  boxes to have the class `information noup` ans this main one `windowbg noup`

![Screenshot_2020-05-05 Privacy Policy](https://user-images.githubusercontent.com/431799/81141422-7520d480-8f21-11ea-9524-3c76438af3bb.png)